ghcide/src/Development/IDE/Core/Compile.hs

```diff
@@ -1039,7 +1039,8 @@ getDocsBatch
   :: HscEnv
   -> Module  -- ^ a moudle where the names are in scope
   -> [Name]
-  -> IO (Either ErrorMessages (Map.Map Name (Either GetDocsFailure (Maybe HsDocString, Maybe (Map.Map Int HsDocString)))))
+  --  2021-11-18: NOTE: Map Int would become IntMap if next GHCs.
```

Comment:
I noted that in the newest GHC docs the type changed to more logical implementation.

Reply:
Now updated to `IntMap`.